src/components/views/messages/TileErrorBoundary.tsx

```diff
@@ -92,8 +94,9 @@ export default class TileErrorBoundary extends React.Component<IProps, IState> {
                 <div className="mx_EventTile_line">
                     <span>
                         { _t("Can't load this message") }
-                        { mxEvent && ` (${mxEvent.getType()})` }
+                        { mxEvent && ` (${mxEvent.getType()})&nbsp;` }
                         { submitLogsButton }
+                        { spaceViewSourceButton }
                         { viewSourceButton }
```

Comment:
```suggestion
                        { _t("Can't load this message") } &nbsp;
                        { mxEvent && `(${mxEvent.getType()})` } &nbsp;
                        { submitLogsButton } &nbsp;
                        { viewSourceButton }
```

I think this would probably be the clearest

Reply:
```
{ submitLogsButton } &nbsp;
```
wouldn't this insert the space even if there is no `viewSourceButton` to follow up?

Reply:
Argh, true - could we move `&nbsp;` into to the if statement?

```
            if (mxEvent && SettingsStore.getValue("developerMode")) {
                viewSourceButton = <>
					&nbsp;
					<AccessibleButton onClick={this.onViewSource} kind="link">
                    	{ _t("View Source") }
              		</AccessibleButton>
				</>;
            }
```

Reply:
That's what I tried in my last change, without awareness of that empty tags syntax you're proposing. Will update and see what the CI says.